ledger-tool/src/output.rs

```diff
@@ -626,19 +626,25 @@ pub struct AccountsScanner {
     account_data_encoding: UiAccountEncoding,
 }
 
+impl AccountsScanner {
```

Comment:
why the separate impl context?

Reply:
yeah the methods fine i was just wondering why it wasn't put under the `impl AccountsScanner` block at ln635. afaik the compiler will collapse them, but a little jarring in the code

Reply:
D'oh; I completely missed your point the first time; I think I just refactored it enough times that I had blinders on what you were calling out. Done in https://github.com/solana-labs/solana/pull/34915/commits/00b3088bbb79550c8ad27455f33dd56fc48fcf05